application/change-organizations.mjs

```diff
@@ -33,17 +33,18 @@ export default async function changeOrganizations (institutions, organizations)
     if (organizations.some(hasName)) {
       const organization = organizations.find(hasName)
       const institutionName = getInstitutionName(institution)
+      const organizationMetaData = getMetadata(organization)
 
       if (
-        institutionName !== getOrganizationDisplayName(organization)) {
+        institutionName !== getOrganizationDisplayName(organization) || organizationMetaData === undefined) {
```

Comment:
I think this will fail to update an organisation's metadata on the second run of the script, maybe? Say, if an iid ever changed?

I feel like we should instead determine if the organisation metadata needs to be updated, instead of checking if it is undefined. In this case, we need to check if the current metadata.institutionId is different than the one we want it to be, so that's the condition. If the current metadata.institutionId is undefined, the condition would still match, so the update would still be performed.

To make it more robust we should actually compare the current metadata and the desired metadata (using only the keys in the desired metadata object). If any changes were detected, then perform the update; otherwise, create the metadata from scratch (the `else`).